test/helpers/oracleHelpers.ts

```diff
@@ -90,32 +92,55 @@ export const deployHistoricalPriceFeedAsync = async(
   );
 };
 
-export const deployHistoricalPriceFeedV2Async = async(
+export const deployTimeSeriesFeedAsync = async(
   web3: Web3,
-  medianizerAddress: Address,
+  dataSourceAddress: Address,
   updateFrequency: BigNumber = ONE_DAY_IN_SECONDS,
-  updateTolerance: BigNumber = ONE_DAY_IN_SECONDS.div(4),
   maxDataPoints: BigNumber = new BigNumber(200),
   dataDescription: string = '200DailyETHPrice',
-  seededValues: BigNumber[] = [],
+  seededValues: BigNumber[] = [new BigNumber(200 * 10 ** 18)],
   from: Address = TX_DEFAULTS.from,
-): Promise<HistoricalPriceFeedV2Contract> => {
-  const truffleHistoricalPriceFeedContract = contract(HistoricalPriceFeedV2);
-  truffleHistoricalPriceFeedContract.setProvider(web3.currentProvider);
-  truffleHistoricalPriceFeedContract.setNetwork(50);
-  truffleHistoricalPriceFeedContract.defaults(TX_DEFAULTS);
+): Promise<TimeSeriesFeedContract> => {
+  const truffleTimeSeriesFeedContract = contract(TimeSeriesFeed);
```

Comment:
@asoong At some point, we should create a helper method or class that consolidates these settings into a single function vs. duplication 4 lines per contract and refactor everything.